### PR TITLE
To Do List: Add & Remove

### PR DESCRIPTION
**Pull request by Kasaija-Kenneth**
- [x] Implement a function for `adding` a new task
- [x] Implement a function for `deleting` a task
- [x] All changes to the To Do List should be saved in `local storage

### CODE REVIEW

File: src/removeToDo.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+/* eslint-disable no-undef */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable import/no-cycle */
+
+import './index.js';
+
+export const removeToDo = (element) => {
+  element.parentNode.parentNode.removeChild(element.parentNode);
+  LIST[element.index].deleted = true;
+};
```
Comment (Rustamxon7) [inline src/removeToDo.js:11]: Let's break down your code.

Main requirements: 
1. A function for adding a new task (add a new element to the array) is implemented. ✅
1. A function for editing task descriptions is implemented. ❌
1. A function for deleting a task (removing an element from the array) is implemented. ([this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Your%20task%20is%20to%20build%20a%20website%20that%20displays%20a%20list%20that%20looks%20and%20behaves%20like%20the%20part%20of%20minimalist%20project%20capture%20in%20the%20video%20below.)) ❌
1. By default, new tasks should have the property `completed` set to `false` and the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). ❌
1. Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. ❌

File: src/style.css [hunk 2]
```diff
@@ -1,73 +1,126 @@
+/* ------------ youtube.com/CodeExplained ------------ */
```
Comment (Rustamxon7) [inline src/style.css:1]: - [ ] A function for editing task descriptions is implemented. ❌ 

> I cannot see any edit option in your project. Kindly implement it.

File: src/style.css [hunk 3]
```diff
@@ -1,73 +1,126 @@
...
 body {
-  background-color: rgb(241, 233, 233);
+  padding: 0;
```
Comment (Rustamxon7) [inline src/style.css:3]: - [ ] A function for deleting a task (removing an element from the array) is implemented. ([this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Your%20task%20is%20to%20build%20a%20website%20that%20displays%20a%20list%20that%20looks%20and%20behaves%20like%20the%20part%20of%20minimalist%20project%20capture%20in%20the%20video%20below.)) ❌

It's removing all items when I click the refresh button. But it's not meant to be. If I click `trashcan` you need to remove item form list as well as from the localstore.

File: src/style.css [hunk 4]
```diff
@@ -1,73 +1,126 @@
...
+  margin: 0;
+  background-color: rgba(0, 0, 0, 0.1);
```
Comment (Rustamxon7) [inline src/style.css:5]: - [ ] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. ❌

If I open the local store it shows me an index of 0 for all your tasks. If I add it also needs to increase:

> add
> task 1 = index 1
> task 2 = index 2 ...

> delete tasks 2 
> remaining tasks index = 1

If I delete the item from the list. It also refreshes the indexes of the tasks.

File: src/style.css [hunk 5]
```diff
@@ -1,73 +1,126 @@
+/* ------------ youtube.com/CodeExplained ------------ */
 body {
-  background-color: rgb(241, 233, 233);
+  padding: 0;
+  margin: 0;
```
Comment (Rustamxon7) [inline src/style.css:4]: - [ ] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). ❌

I cannot make your task complete or incomplete.

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hello @Kasaija-Kenneth, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great Design_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @Kasaija-Kenneth, 

# STATUS: COMPLETE 😃

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

No linter errors ✔️
Correct Github flow ✔️
Descriptive PR ✔️
Professional README file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**